src/appservice-kube/azext_appservice_kube/_params.py

```diff
@@ -93,7 +93,7 @@ def load_arguments(self, _):
                    completer=get_resource_name_completion_list('Microsoft.Web/serverFarms'),
                    help="name or resource id of the function app service plan. Use 'appservice plan create' to get one")
         c.argument('new_app_name', options_list=['--name', '-n'], help='name of the new function app')
-        c.argument('custom_location', help="Name or ID of the custom location")
+        c.argument('custom_location', help="Name or ID of the custom location. Use an ID for a custom location in a different resource group from the app")
```

Comment:
can we update help.py with an example for using custom-location ID in a different RG so the usage understood?